setup.py

```diff
@@ -58,6 +58,7 @@
         "django-oauth-toolkit<2.0.0",
         "django-object-actions",
         "django-spurl",
+        "django-storages>=1.14"
```

Comment:
This should always have been a direct dependency (not just relying on it it transitively existing via `django-s3-file-field[boto3]`), since we import it directly within this project.